code/modules/awaymissions/_awaymission.dm

```diff
@@ -0,0 +1,192 @@
+//#define DISABLE_AWAYMISSIONS_ON_ROUNDSTART
+
+var/global/list/datum/away_mission/existing_away_missions = list()
+
+var/global/list/awaydestinations = list() //List of landmarks
+/obj/effect/landmark/awaystart
+	name = "awaystart"
+/*
+There are two ways to add an away mission to the game
+
+ A) Add the map file's location to maps/RandomZLevels/fileList.txt
+ B) Create a subtype of /datum/away_mission and set its file_dir to the map file's location (see below)
+
+First method sucks
+
+Second method lets you use the initialize() proc to interact with the away mission after it has loaded (for example generate map elements, create shuttles, etc).
+It also lets you write a description for the away mission, as well as many other things
+
+Example of the second method:
+
+/datum/away_mission/my_shit
+	file_dir = "maps/RandomZLevels/fresh_dump.dmm"
+	desc = "This stinks"
+
+****READ THIS****
+  Because a z-level is 500x500 in size, loading an away mission creates 250,000 new turfs - in addition to any additonal mobs and objects.
+
+  If your away mission is smaller than 500x500, its northern and eastern borders will be surrounded with space turfs. Unless you're fine with this, you
+ should secure these borders with an indestructible wall (insert a trump meme here) so that nobody can get out
+
+
+*/
+
+/datum/away_mission
+	var/name = "" //Name of the mission
+	var/file_dir = "" //Path of the file. Example: "maps/RandomZLevels/test.dmm"
```

Comment:
Honestly this var name sounds like it references a directory instead of a file path.